src/graphql/uploadSiloDiscoveryResults.ts

```diff
@@ -0,0 +1,32 @@
+import { chunk } from 'lodash';
+import { mapSeries } from 'bluebird';
+import { ADD_SILO_DISCOVERY_RESULTS } from './gqls';
+import { GraphQLClient } from 'graphql-request';
+import { SiloDiscoveryRawResults } from '../plugins';
+
+const CHUNK_SIZE = 1000;
+
+/**
+ * Uploads silo discovery results for Transcend to classify
+ *
+ * @param client - GraphQL Client
+ * @param pluginId - pluginID to associate with the results
+ * @param results - The results
+ */
+export async function uploadSiloDiscoveryResults(
+  client: GraphQLClient,
+  pluginId: string,
+  results: SiloDiscoveryRawResults[],
+): Promise<void> {
+  const chunks = chunk(results, CHUNK_SIZE);
+
+  await mapSeries(chunks, async (rawResults: SiloDiscoveryRawResults[]) => {
```

Comment:
```suggestion
  await mapSeries(chunks, async (rawResults) => {
```

should be able to infer